groupby/perfect_groupby.cpp

```diff
@@ -91,7 +91,8 @@ void PerfectGroupBy::_run(const size_t buf_size, Meter &meter) {
        h.single_task<class clean>([=]() {
          for (size_t i = 0; i < groups_count * threads_count; i++) {
            ht_v[i] = 0;
-           o[i] = 0;
+           if (i < groups_count)
+             o[i] = 0;
```

Comment:
What's the logic behind this one?

Reply:
Here we want to clear all output from previous iteration using `single_task`
However, `groups_count * threads_count` may be bigger than output array size, so we need to control it to prevent `SEGFAULT`
This is not so clear, I wanted to do it in one for loop, but is it better to separate this two operations into two `single_task`s?

Reply:
I guess no big difference since it's a single task